uoftblueprint/merit-award
# JWT Auth

Use simple jwt library to add authenticated endpoints to our API.

Testing Steps: 
1. Create a user with the endpoint `/api/users/create`
2. Login as a user and you should get a access token and a refresh token.
3. Try and access `/api/users` without the access token. You should get a 401 error.
4. Add the access token as a bearer token and access the same route. This should return the user.
5. Access `/api/refresh`. You should get a new access token.
